src/main/java/com/ylysenkova/movieland/service/impl/MovieServiceImpl.java

```diff
@@ -22,13 +32,29 @@
 
     @Override
     public List<Movie> getThreeMovies() {
-        return movieDao.getThreeMovies(movieDao.getThreeMovieIds());
+        List<Movie> enrichedMovie = new ArrayList<>();
+        enrichedMovie.addAll(movieDao.getThreeMovies(movieDao.getThreeMovieIds()));
+        genreService.enrichMovieWithGenres(enrichedMovie);
+        countryService.enrichMovieWithCountries(enrichedMovie);
+        return enrichedMovie;
     }
 
     public List<Movie> getMovieByGenreId(int genreId) {
         return movieDao.getMovieByGenreId(genreId);
     }
 
+    @Override
+    public Movie getMovieById(int movieId) {
+        Movie movieForEnrichment = movieDao.getMovieById(movieId);
+        List<Movie> enrichedMovieWithReview = new ArrayList<>();
```

Comment:
Why do you create unneeded list of movies instead of create additional method to enrich single movie by countries and genres like enrichSingleMovieWithGenres(Movie movie)